backward.hpp

```diff
@@ -927,7 +922,18 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libbfd>
       return trace;
     }
 
-    trace.object_filename = symbol_info.dli_fname;
+    if(symbol_info.dli_fname == get_argv0()) {
```

Comment:
resolve is called in a loop when a stacktrace is resolved. Please cache get_argv0 with an attribute. Similar to `_demangler`.

(this is an improvement to the previous code)